src/webapi/Service/PeriodicHostedSyncOdsAssetsService.cs

```diff
@@ -0,0 +1,57 @@
+namespace eppeta.webapi.Service
+{
+    class PeriodicHostedSyncOdsAssetsService : BackgroundService
+    {
+        private readonly TimeSpan _period = TimeSpan.FromHours(AppSettings.SyncOdsAssetsSettings.PeriodInHours);
+        private readonly ILogger<PeriodicHostedSyncOdsAssetsService> _logger;
+        private readonly IServiceScopeFactory _factory;
+        private long _executionCount = 0;
+        public bool IsEnabled { get; set; } = true;
+
+        public PeriodicHostedSyncOdsAssetsService(ILogger<PeriodicHostedSyncOdsAssetsService> logger, IServiceScopeFactory factory)
+        {
+            _logger = logger;
+            _factory = factory;
+        }
+
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            // ExecuteAsync is executed once and we have to take care of a mechanism ourselves that is kept during operation.
+            // To do this, we can use a Periodic Timer, which, unlike other timers, does not block resources.
+            // But instead, WaitForNextTickAsync provides a mechanism that blocks a task and can thus be used in a While loop.
+            using PeriodicTimer timer = new PeriodicTimer(_period);
+
+            // When ASP.NET Core is intentionally shut down, the background service receives information
+            // via the stopping token that it has been canceled.
+            // We check the cancellation to avoid blocking the application shutdown.
+            while (
+                !stoppingToken.IsCancellationRequested &&
+                await timer.WaitForNextTickAsync(stoppingToken))
+            {
+                if (IsEnabled)
```

Comment:
What about extracting this code block (everything in the while loop) to a different method. And that method can be called in the while loop, and before the loop as well. 
We can review this alternative tomorrow.